datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -2480,6 +2480,399 @@ Select bit_xor(DISTINCT c), arrow_typeof(bit_xor(DISTINCT c))  from t;
 statement ok
 drop table t;
 
+#################
+# Min_Max Begin #
+#################
+statement ok
```

Comment:
I think generally it looks good to me. But could you please add some comments to the sqllogictest?
For each unit test, add a comment beginning with `# `

Reply:
It is nice to have. I fine if there is no comment for trivial test

Reply:
Done.